src/mongo/delegates/Views.class.php

```diff
@@ -571,9 +571,9 @@ protected function doJoins($source, $joins, &$dest, $from, $contextAlias, $build
                 {
                     // single value for join
                     $joinUris[] = array(_ID_RESOURCE=>$source[$predicate][VALUE_URI],_ID_CONTEXT=>$contextAlias);
-                }
-                else
-                {
+                } else if($predicate == '_id') {
+                    $joinUris[] = array(_ID_RESOURCE => $source[$predicate][_ID_RESOURCE], _ID_CONTEXT => $contextAlias);
```

Comment:
Since Tripod now requires PHP 7.3+, we can ditch `array(...)`, but I appreciate that consistency may be more important